services/api-server/src/simcore_service_api_server/services/webserver.py

```diff
@@ -409,6 +411,21 @@ async def connect_pricing_unit_to_project_node(
             )
             response.raise_for_status()
 
+    async def start_project(
+        self, project_id: UUID, cluster_id: ClusterID | None = None
+    ) -> None:
+        with _handle_webserver_api_errors():
+            body_input: dict[str, Any] = {}
+            if cluster_id:
+                body_input["cluster_id"] = cluster_id
+            body: ComputationStart = ComputationStart(**body_input)
+            response = await self.client.post(
+                f"/computations/{project_id}:start",
+                cookies=self.session_cookies,
+                json=jsonable_encoder(body),
```

Comment:
TIP: Note that we are using the same model in the webserver therefore you can spare sending the defaults if you like
```python
json = jsonable_encode(body, exclude_unset=True, exclude_defaults=True)
```

NOTE: This is an optimization that we should do at some point systematically when we have the models really in sync

Reply:
Good point. I will correct